bot/cogs/math.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+
+from io import BytesIO
+from urllib.parse import quote
+
+
+from aiohttp import ClientSession
+
+from discord import File
+from discord.ext.commands import command
+
+from sympy import latex
+from sympy.parsing.sympy_parser import parse_expr
+
+
+LATEX_URL = "https://latex.codecogs.com/png.download?%5Cdpi%7B300%7D%20%5Cbg_white%20%5Chuge%20"
```

Comment:
Consider removing `bg_white` from the query string completely, to return an expression with a transparent background. You might also want to decrease that text size a little. `150` seems to work well in discord.

Reply:
bg_white works better for users on dark or custom themes, I will change the text size though